script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -88,7 +88,7 @@ She must be. I'm like a deer caught in the headlights.
 [sha:9a3f3b9ab5defb58ba9260111dde892a605ef35c]{
 -- Page 56, Offset 14058.
 -- やっぱりこのまま適当に挨拶をして撤退するのが良策なのか。
-Perhaps it would be a good idea to just respond with an appropriate greeting and then retreat.
+Perhaps it'd be a good idea to just respond with an appropriate greeting and retreat afterwards.
```

Comment:
contraction fine, but might revert the other change. Hard to explain why, but it reads a bit more like maybe the retreat was mentioned prior and is now being assigned a time.